Kickstarter-iOS/Features/DesignSystem/Colors/DesignSystemCoreColors.swift

```diff
@@ -42,18 +49,115 @@ public enum DesignSystemColors: String {
   case facebookBlue
   case inform
   case warn
+
+  // MARK: NEW COLORS
+
+  // MARK: Blues
+
+  case blue1
+  case blue2
+  case blue3
+  case blue4
+  case blue5
+  case blue6
+  case blue7
+  case blue8
+  case blue9
+  case blue10
+
+  // MARK: Greens
+
+  case green1
+  case green2
+  case green3
+  case green4
+  case green5
+  case green6
+  case green7
+  case green8
+  case green9
+  case green10
+
+  // MARK: Greys
+
+  case grey1
+  case grey2
+  case grey3
+  case grey4
+  case grey5
+  case grey6
+  case grey7
+  case grey8
+  case grey9
+  case grey10
+
+  // MARK: Oranges
+
+  case orange1
+  case orange2
+  case orange3
+  case orange4
+  case orange5
+  case orange6
+  case orange7
+  case orange8
+  case orange9
+  case orange10
+
+  // MARK: Purples
+
+  case purple1
+  case purple2
+  case purple3
+  case purple4
+  case purple5
+  case purple6
+  case purple7
+  case purple8
+  case purple9
+  case purple10
+
+  // MARK: Reds
+
+  case red1
+  case red2
+  case red3
+  case red4
+  case red5
+  case red6
+  case red7
+  case red8
+  case red9
+  case red10
+
+  // MARK: Yellows
+
+  case yellow1
+  case yellow2
+  case yellow3
+  case yellow4
+  case yellow5
+  case yellow6
+  case yellow7
+  case yellow8
+  case yellow9
+  case yellow10
 }
 
-extension DesignSystemColors {
-  public func load() -> UIColor {
-    UIColor(named: self.rawValue) ?? .white
+extension DesignSystemCoreColors {
+  public func load(_ colorSet: DesignSystemColorSet) -> UIColor {
+    UIColor(named: "\(colorSet.rawValue)/\(self.rawValue)") ?? .white
   }
 }
 
-public func adaptiveColor(_ style: DesignSystemColors) -> UIColor {
-  style.load()
+public func adaptiveColor(_ colorSet: DesignSystemColorSet, _ style: DesignSystemCoreColors) -> UIColor {
+  style.load(colorSet)
 }
 
+// MARK: Component Styles
+
+/// New component style examples.
+/// Essentially a copy/paste of how we're currently using Prelude to set component styles. These just take advantage of the new adaptiveColor pattern above.
```

Comment:
A question that is extremely out of scope for this particular PR: what would it look like to move the new design system off of Prelude?

Reply:
It would look like a lot of work. Ideally, that happens as we integrate SwiftUI.

Reply:
If we do ultimately want to move off Prelude, I do think this would be a good time to start thinking about it. (For example, we'd presumably pick a different way of modeling these than using `.template` everywhere.) Like I said, out of scope here, but now feels like as good a time as any to fix it.

One thing I've been thinking about for the SwiftUI transition - we're going to want to fix the app infrastructure before we worry about back-porting individual screens. Design systems are part of that infrastructure, which is why I'm bringing it up now.